Adding option to pass  publish target

jbosse opened the pull request:
See https://developer.chrome.com/webstore/using_webstore_api#trustedtesters

jbosse commented:
I will have to figure out what I did wrong. :-(

trmcnvn commented on src/main.ts:
```diff
@@ -61,16 +61,17 @@ async function run() {
     const refresh = core.getInput('refresh-token', { required: true });
     const zip = core.getInput('zip', { required: true });
     const extension = core.getInput('extension');
+    const publishTarget = core.getInput('publish-target') || "default";
```
You need to define `publish-target` in the `action.yml` file and set the default value there.

trmcnvn replied:
You can provide a default in the action.yml file

```yaml
default: default
```

jbosse commented:
Thanks for the feedback!

-Jimmy
On Mar 21, 2020, 3:44 PM -0400, Thomas McNiven <notifications@github.com>, wrote:
> @trmcnvn requested changes on this pull request.
> In src/main.ts:
> > @@ -61,16 +61,17 @@ async function run() {
>     const refresh = core.getInput('refresh-token', { required: true });
>     const zip = core.getInput('zip', { required: true });
>     const extension = core.getInput('extension');
> +    const publishTarget = core.getInput('publish-target') || "default";
> You need to define publish-target in the action.yml file and set the default value there.
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub, or unsubscribe.

jbosse commented:
I have implemented the missing `action.yml` change and cleaned up the whole PR to just include the files I really changed. Is there a reason to have `node_modules` in the repo?

trmcnvn commented:
> I have implemented the missing `action.yml` change and cleaned up the whole PR to just include the files I really changed. Is there a reason to have `node_modules` in the repo?

Yes, the `node_modules` folder is required for the action to be used as a workflow won't install dependencies itself.

jbosse commented:
Thanks for being so patient.

trmcnvn commented:
Thanks! 👍
